vuu-ui/packages/vuu-layout/src/layout-persistence/RemoteLayoutPersistenceManager.ts

```diff
@@ -0,0 +1,190 @@
+import {
+  ApplicationLayout,
+  LayoutMetadata,
+  LayoutMetadataDto,
+} from "@finos/vuu-shell";
+import { LayoutPersistenceManager } from "./LayoutPersistenceManager";
+import { LayoutJSON } from "../layout-reducer";
+
+const DEFAULT_SERVER_BASE_URL = "http://127.0.0.1:8081/api";
+
+const baseURL = process.env.LAYOUT_BASE_URL ?? DEFAULT_SERVER_BASE_URL;
```

Comment:
need entry for this process.env in main build (esbuild.mjs)

Reply:
Also covered by [this ticket](https://github.com/ScottLogic/finos-vuu/issues/113).

Reply:
Done! [9d060cf](https://github.com/finos/vuu/pull/970/commits/9d060cfa76f0abb7f2aefd5e1c3ff93c1fbbf9d8)